client/src/main/java/org/apache/celeborn/client/read/ChunkClient.java

```diff
@@ -149,13 +144,11 @@ public synchronized int openChunks() throws IOException {
     if (numChunks == -1) {
       if (currentException != null) {
         throw new IOException(
-            String.format(
-                "Could not open chunks from %s after %d tries.", currentReplica, numTries),
+            String.format("Could not open chunks from %s after %d tries.", replica, numTries),
             currentException);
       } else {
         throw new IOException(
-            String.format(
-                "Could not open chunks from %s after %d tries.", currentReplica, numTries));
+            String.format("Could not open chunks from %s after %d tries.", replica, numTries));
       }
     }
```

Comment:
We should set numTries to zero before return

Reply:
done